fuel-core/src/database.rs

```diff
@@ -566,5 +566,51 @@ mod relayer {
                     );
                 });
         }
+
+        async fn get_pending_committed_fuel_height(&self) -> Option<BlockHeight> {
```

Comment:
Should we just make this method fallible instead of panicking?

Reply:
I was just copying the behavior of all the other trait methods. I'm not really sure what usecases can actually return an error but if there are some then we should probably change all the functions on this trait to be fallible in a new PR?